apps/src/sites/studio/pages/maker/discountcode.js

```diff
@@ -26,23 +25,15 @@ $(document).ready(() => {
 
   ReactDOM.render(
     <div>
-      {!scriptData.discountCode &&
-        <EligibilityChecklist
-          statusPD={scriptData.is_pd_eligible ? Status.SUCCEEDED : Status.FAILED}
-          statusStudentCount={scriptData.is_progress_eligible ? Status.SUCCEEDED : Status.FAILED}
-          unit6Intention={scriptData.unit_6_intention}
-          schoolId={scriptData.school_id}
-          schoolName={scriptData.school_name}
-          hasConfirmedSchool={scriptData.has_confirmed_school}
-          getsFullDiscount={scriptData.gets_full_discount}
-        />
-      }
-      {scriptData.discountCode &&
-        <DiscountCodeInstructions
-          discountCode="XXXXXX"
-          fullDiscount={true}
-        />
-      }
+      <EligibilityChecklist
+        statusPD={scriptData.is_pd_eligible ? Status.SUCCEEDED : Status.FAILED}
+        statusStudentCount={scriptData.is_progress_eligible ? Status.SUCCEEDED : Status.FAILED}
+        unit6Intention={scriptData.unit_6_intention}
+        schoolId={scriptData.school_id}
+        schoolName={scriptData.school_name}
+        hasConfirmedSchool={scriptData.has_confirmed_school}
+        getsFullDiscount={scriptData.gets_full_discount}
+      />
```

Comment:
I moved `DiscountCodeInstructions` into `EligibilityChecklist` so that we can easily show the instructions after completing the dialog.